### DISCUSSION TITLE
Statistics Card - Option to hide legend

### DISCUSSION THREAD

**Discussion by alexpomsft** [home-assistant/frontend#9771] [Dashboard]
When plotting three or more statistics the legend can get overwhelming when also displaying min, max, mean, etc. It would be great if we could hide the legend. Alternatively it would be great show the legend but only have one item per statistic added, e.g. just [Kitchen, Living Room, Basement] in the below image.

![image](https://user-images.githubusercontent.com/6499675/128927392-2d8da6aa-2f1a-4420-a3ed-708ab0be7bc8.png)

**Comment by karwosts** [comment]
Same as https://github.com/home-assistant/frontend/discussions/9846  (should be resolved now as well)
